experiments/axon_geometry/scripts/generate_trace_data.py

```diff
@@ -45,11 +56,65 @@ def generate_brain_trace_data(brain: str, spacing: int):
             print("Initialized GeometricGraph")
             spline_tree = G.fit_spline_tree_invariant()
             print("Computed splines")
+            
```

Comment:
Why is all this code commented? If you want this code to stay, is it possible to put it inside a conditional? If its not necessary, can you remove?

Reply:
Not necessary, removed. It was the code I used to plot the dendograms of the spline trees